src/components/DefaultTable.vue

```diff
@@ -138,14 +144,22 @@ export default {
 
       if (header.format) {
         return header.format(cellValue)
-      } else if (header.isDate) {
-        if (!cellValue) return null
+      } else if (cellValue === null || cellValue === undefined) {
+        return this.emptyDisplay
+      } else if (header.type === 'date') {
         return new Date(cellValue).toLocaleDateString(this.locale)
+      } else if (header.type === 'currency') {
+        const value = parseInt(cellValue)
+        return currencyFormatter.format(value, {
+          code: row.currency || null
+        })
       } else {
         return cellValue
       }
-    }
+    },
+    getCurrencyFormat (amount) {
```

Comment:
Is there a specific reason why you kept this empty function? It's unused as well

Reply:
nope. I think it was me going to do something cleaver then not, and not cleaning up my code. Gone now